internal/services/cdn/cdn_frontdoor_helpers.go

```diff
@@ -104,17 +105,24 @@ func flattenFrontDoorTags(tagMap map[string]*string) *map[string]string {
 
 func flattenTransformSlice(input *[]frontdoor.TransformType) []interface{} {
 	result := make([]interface{}, 0)
+	if len(*input) == 0 || input == nil {
```

Comment:
this will panic if `input` is nil in this order

```suggestion
	if input == nil || len(*input) == 0 {
```